### PR TITLE
Add option to force close channel or abandon channel

### PR DESCRIPTION
**Pull request by benthecarman**
This should help with issues like @shyfire131 had where he has a channel that will never open.

### CODE REVIEW

File: mutiny-core/src/nodemanager.rs [hunk 1]
```diff
@@ -1765,17 +1765,34 @@ impl<S: MutinyStorage> NodeManager<S> {
...
-                            self.logger,
-                            "had an error closing channel {} with node {} : {e:?}",
-                            &channel.channel_id.to_hex(),
-                            &channel.counterparty.node_id.to_hex()
-                        );
-                        MutinyError::ChannelClosingFailed
-                    })?;
+                if force {
+                    node.channel_manager
+                        .force_close_broadcasting_latest_txn(
```
Comment (TonyGiorgio) [inline mutiny-core/src/nodemanager.rs:1770]: THe issue where the channel opening will never confirm, should that be by force closing without broadcasting latest txn?

Comment (benthecarman) [reply mutiny-core/src/nodemanager.rs:1770]: hmm true, maybe add an `abandon_channel` function that does that

Comment (benthecarman) [reply mutiny-core/src/nodemanager.rs:1770]: added

Comment (TonyGiorgio) [reply mutiny-core/src/nodemanager.rs:1770]: What do you think of having `force` and `abandon` as a parameter here? The code is pretty similar.

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**